sdk/communication/AzureCommunicationCommon/CHANGELOG.md

```diff
@@ -1,9 +1,14 @@
 # Release History
 
-## 1.2.0 (upcoming)
+## 2.0.0-beta.1 (upcoming)
 ### Features Added
 - Optimization added: When the proactive refreshing is enabled and the token refresher fails to provide a token that's not about to expire soon, the subsequent refresh attempts will be scheduled for when the token reaches half of its remaining lifetime until a token with long enough validity (>10 minutes) is obtained.
 - Added `cancel()` to `CommunicationTokenCredential` that cancels any internal auto-refresh operation. An instance of `CommunicationTokenCredential` cannot be reused once it has been canceled, otherwise the error will be returned.
+- Added support for a new communication identifier `MicrosoftBotIdentifier`.
+
+### Breaking Changes
+- Introduction of `MicrosoftBotIdentifier` is a breaking change. It will affect code that relied on using `UnknownIdentifier` with a rawID starting with `28:`
```

Comment:
Yeah, it complete defeats the purpose of extensible enums if adding a new value is still considered a breaking change. We cannot do a major version bump every time a new identifier is added. Let's set up a time to discuss. Please include @mpodwysocki